### ISSUE TITLE
Enhancement of warp helper/maneuver node editor (stock orbits)

### ISSUE THREAD

**Issue by Kerbinator-Fras** [MuMech/MechJeb2#1854]
1. **allow specifying lead time per-level of warp**
(for example, after this level is met, wait 1 sec and go to next level.
current lead time is 0s when warp rate goes up and ~5s (?) when warp rate goes down, not counting the time of adjustment
and **add a hard limit of warp rate (in form of level, not multiplyer, adjustable in BTW). in order for it not to warp too fast**)
!Extra!: note that this is a KSP problem in 1.8+.
in 1.8+ there are two glitches in orbit prediction system
(1) in 1:1 resonant orbit (like Kerbin-Mun-Kerbin-Mun), the encounter will be at after 1k years when it actually is 1 year
this looks like not that likely to happen pre-1.8, and happens only when you are going K-K start inside end outside
this is non-fatal and can be solved by adding empty nodes.
(2) in deep retrograde orbit (like you enter Kerbin SOI retrograde and have a ~30deg angle of hyperbolic trajectory)
so you can encounter Mun when you go in or you go out, sometimes the go-in one will disappear causing it go wrong
it won't even appear when you find your ship is going straight to Mun until it suddenly go into Mun's SOI
and the game finally realized it's doing wrong.
this is fatal but only happen within a certain range. by carefully adjusting to avoid that range you sacrifice 1dv to get rid of it.
best if both bugs fixed, the orbit encounter bugs fixing system. 
2. **direct UT display/edit for nodes**
so then you just don't need a additional precise node mod then, I'm only using that for node UT input
(copy nodes between savefiles etc. that is useful, also precise node itself have a problem:
precise node update every time you click your key. so for example your node is 10000s and 10015s
and you want the second one to be 10016, then you backspace making it 1001 then it become the first node that mess everything
MechJeb won't do that because your input is valid after and only after you click "update" button, actually pretty good
3. MJ execution pointing problem
there is a known thing that (without principia/persistentRotation) when on-rail the vessel's orientation is:
fixed, in main space; and rotate at planet's angular speed, in rotated planetarium
this cause when you are under invertRotAltitudeThreshold, your ship will drift away from node
MJ re-correct your ship 8 minutes before ignition, kinda interrupting for filming. also stock ship usually have strong SAS.
now I use manual "warp helper" and start the burn before the stock ignition counter go to 13s (my ship only need 5s to adjust.)
so it will be good to give an option of whether you
**1)point the ship to node just before executing (also able to specify lead time)
2)or when you press execute immediately (just like what is been doing now)**
also, another thing is to add an option:
**when your node's left dv is lower than a certain amount (for example 1e-4), stop vessel from rotating**
it is known that when your node's left dv is lower than 1e-5, KSP will not give out node dv info anymore.
this makes your ship turn around fast and ruin your execution
(as stock SAS is very strong, and change in angular vel will change your actual speed in stock KSP).
currently I do this by pressing P in AA when node is getting to 0.0, as AA have higher control authority stop vessel from spinning.
by overusing this I get node burn precision to amazing 1e-5 which is incredibly good for gravity assist kits.

note: this is mainly for filming thingy, as you want to keep things as stable as posssible.
I'm not too concerned about takeoff/landing of my SSTO since I fly that manually and quite tricky
but precisely executing node is important for me, and I'm glad that MechJeb have it, written in hope for better.
